doc/release-notes/roadmap.en.rst

```diff
@@ -27,7 +27,7 @@ ATS Release Roadmap
 
 Apache Traffic Server is currently on a two year major release cycle. At any
 given time, we guarantee to have two major versions supported. Currently,
-this means v7.1.x and v8.0.x, with a possible v8.1.x coming.
+this means v8.1.x and v9.2.x, with a possible v10.x.x coming.
```

Comment:
`v8.x` and `v9.x` is better? The current latest version is 9.1.2, but 9.2.0 is coming soon.

Reply:
The recommendation from our PR and issue scrub was to state something that doesn't need to be updated.  "Please refer to our download page to see the current supported versions."  and link to the download page.